tests/integration/targets/latest/tasks/controllers.yml

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
These tests won't run until you add an entry to the [main.yml](https://github.com/nautobot/nautobot-ansible/blob/develop/tests/integration/targets/latest/tasks/main.yml) file.

Also, since controllers were added into 2.2, you're going to need a block statement around all these tasks like I did for Contacts:
https://github.com/nautobot/nautobot-ansible/blob/900395a17d199fe1816a887fee4cc7992a01e56a/tests/integration/targets/latest/tasks/contact.yml#L148-L150